Python-Home-Challenges/recurring word.py

```diff
@@ -18,5 +14,10 @@
 for key in list(words_in_text.keys()):
     print(key, ":", words_in_text[key])
 print("---------------")
-print (max(words_in_text.values()))
-
+max1=0
+for highest in list(words_in_text.keys()):
+    if max1 < words_in_text[highest]:
+        max1 = words_in_text[highest]
+        key1 = highest
+print ("The most recurring word is: ", key1," with ",max1, " times")
+my_file.close()
```

Comment:
you should close file as soon as you done with reading it, or better - use "with...open..."

Reply:
Changed to with open.